test/schema_parser_test.py

```diff
@@ -796,7 +796,7 @@ def test_parsed_start(self, model, assertion, data):
             (brewtils.models.Event, lazy_fixture("event_dict")),
             (brewtils.models.Queue, lazy_fixture("queue_dict")),
             (brewtils.models.Principal, lazy_fixture("principal_dict")),
-            (brewtils.models.Role, lazy_fixture("role_dict")),
+            (brewtils.models.LegacyRole, lazy_fixture("role_dict")),
```

Comment:
It didn't really matter, but I went through and renamed it to `legacy_role_dict` just to keep it consistent and so that if I end up doing something similar down the line for the new role stuff the `role_dict` name is already free.